libr/include/r_bin.h

```diff
@@ -271,6 +271,8 @@ typedef struct r_bin_section_t {
 	// per section platform info
 	const char *arch;
 	int bits;
+	// when true means the section is mapped in the Virtual Space
```

Comment:
This is probably not the best solution, but another solution would require big changes imo.
The issue is that sections are not what you map, usually (for example in ELF). Segments are what are mapped from the file to the virtual space. So, because we have program headers stored as sections, only program headers have the is_allocatable value to TRUE, while normal sections have FALSE.